Announce 2022 CFP Office Hours (#335)
SeaGL/seagl.github.io: _posts/2022-07-22-office-hours.md

---
layout: post
title: 'SeaGL 2022 CFP Office Hours'
status: publish
type: post
published: true
categories: news
tags: '2022'
---

Our goal is for SeaGL to be a welcoming place for first time (and veteran) speakers. We are therefore pleased to announce the launch of SeaGL 2022 [CFP](https://seagl.org/news/2022/07/06/CFP-Live.html) Office Hours! Office Hours are designed to provide an opportunity for prospective speakers to receive real time assistance with their talks and/or CFP submissions.

> Are you unsure which of your two talk ideas to submit (hint: submit both!)? Do you need help crafting an abstract for your talk? Or perhaps you are interested in guidance on how to adapt an existing talk you have presented for a SeaGL audience. We will have members of the SeaGL Program Committee on hand to help answer these and any other questions you might have.

Reviewer: > Or perhaps you are interested in guidance on how to adapt an existing talk you have presented for a SeaGL audience.

Non-native speakers might struggle with this sentence.  How about 'Or perhaps you are interested in adapting a talk that you've presented before so that it fits SeaGL's theme'?

Author: ```suggestion
Are you unsure which of your two talk ideas to submit (hint: submit both!)? Do you need help crafting an abstract for your talk? Or perhaps you are interested in adapting a talk you've presented before so that it fits SeaGL's theme. We will have members of the SeaGL Program Committee on hand to help answer these and any other questions you might have.
```

> Office Hours will be taking place via chat on [Matrix](https://matrix.to/#/#office-hours:seagl.org) (#office-hours:seagl.org) and [IRC](https://web.libera.chat/?channel=#seagl-office-hours) (#seagl-office-hours on Libera.Chat) or on [video](https://meet.seattlematrix.org/SeaGL_Office_Hours) at the following times (Times are in PDT / UTC-7):

Reviewer 2: For the web chat this also works, in case you’d prefer it: https://irc.seagl.org/?channel=#seagl-office-hours

Author: ```suggestion
Office Hours will be taking place via chat on [Matrix](https://matrix.to/#/#office-hours:seagl.org) (#office-hours:seagl.org) and [IRC](https://irc.seagl.org/?channel=#seagl-office-hours) (#seagl-office-hours on Libera.Chat) or on [video](https://meet.seattlematrix.org/SeaGL_Office_Hours) at the following times (Times are in PDT / UTC-7):
```
